packages/vitest/src/node/types/browser.ts

```diff
@@ -236,6 +300,27 @@ export interface BrowserConfigOptions {
    * @default 30000
    */
   connectTimeout?: number
+
+  expect?: {
```

Comment:
These will have to go somewhere else I believe, it was just to make TypeScript happy for now.